includes/block-patterns/lesson/templates/default.php

```diff
@@ -10,8 +10,32 @@
 }
 
 ?>
-<!-- wp:paragraph {"placeholder":"<?php esc_html_e( 'Write lesson content...', 'sensei-lms' ); ?>","className":"sensei-content-description"} -->
-<p class="sensei-content-description"></p>
+<!-- wp:heading {"className":"sensei-content-title"} -->
```

Comment:
Since it's the default one (used even if they close the modal), I'm wondering if it would be annoying having to delete the content every time they create a new lesson, in case they decide not to use the wizard.

I'm thinking now about 2 options to avoid this.

1. Just remove this template from the list, so we don't have the ugly thumbnail.
2. The option 1 + add a new one with this content, in case we want to offer an option only with text as an example.

WDYT?

Reply:
Yeah, agree with you that this change might not be necessary.

I did it before I fixed the CSS issue. Now, with the full background is much better.

What do you think about keeping the Default layout empty but with the whole background or removing it entirely?

See:
| Before | After |
|--------|--------|
| <img width="386" alt="Screenshot 2025-03-31 at 12 30 40" src="https://github.com/user-attachments/assets/44edc77f-de08-45f3-a02c-0f44d305811c" /> | <img width="388" alt="Screenshot 2025-03-31 at 12 27 51" src="https://github.com/user-attachments/assets/b6d9d641-ac72-44e5-a850-e46f43bf350e" /> | 

<img width="839" alt="Screenshot 2025-03-31 at 12 31 48" src="https://github.com/user-attachments/assets/21d937c4-64d9-40db-b727-eb41989818ff" />